### PR TITLE
Add tests for calling filter block when output returns empty

### PR DESCRIPTION
**Pull request by oliverholworthy**
### Goals :soccer:

Improve testing of Filter block. 

### Implementation Details :construction:

Add test for calling filter block when output returns empty dict.

### CODE REVIEW

File: tests/unit/tf/blocks/core/test_tabular.py [hunk 1]
```diff
@@ -1,28 +1,42 @@
-import merlin.models.tf as ml
```
Comment (oliverholworthy) [inline tests/unit/tf/blocks/core/test_tabular.py:1]: i understand that `mm` is now the preferred import alias, and when we're changing imports as we go.

File: tests/unit/tf/blocks/core/test_tabular.py [hunk 2]
```diff
@@ -1,28 +1,42 @@
...
+    assert mm.Filter([])({}) == {}
+    assert mm.Filter([])(tf_con_features) == {}
+    assert mm.Filter(["unknown"])(tf_con_features) == {}
+
+    assert mm.Filter(Schema())({}) == {}
+    assert mm.Filter(Schema())(tf_con_features) == {}
+    assert mm.Filter(Schema(["unknown"]))(tf_con_features) == {}
+
+    assert mm.Filter(Tags.CONTINUOUS)({}) == {}
+    assert mm.Filter(Tags.CONTINUOUS).set_schema(Schema(["unknown"]))(tf_con_features) == {}
```
Comment (oliverholworthy) [inline tests/unit/tf/blocks/core/test_tabular.py:22]: This tag version is a bit different to the others because passing a Tag to the constructor doesn't provide enough information to figure out the column names.

File: tests/unit/tf/blocks/core/test_tabular.py (hunk 1 above)
Comment (rnyak) [reply tests/unit/tf/blocks/core/test_tabular.py:1]: Yes, we go with `mm`.

**Review by gabrielspmoreira — APPROVED** [review]
